sys/src/9/amd64/main.c

```diff
@@ -541,9 +566,9 @@ main(uint32_t mbmagic, uint32_t mbaddress)
 	active.exiting = 0;
 	active.nbooting = 0;
 
-	asminit();
-	multiboot(mbmagic, mbaddress, 0);
-	options(oargc, oargv);
+	asminit(); post("	asminit();", postterminal++);
+	multiboot(mbmagic, mbaddress, 0); post("	multiboot(mbmagic, mbaddress, 0);", postterminal++);
```

Comment:
What's up with the tabbing here? Intentional?

Reply:
yeah, it's so via grep I can look for only those lines.